sal_interfaces/ATPneumatics/ATPneumatics_Commands.xml

```diff
@@ -89,13 +89,13 @@
         <Property/>
         <Action/>
         <Value/>
-        <Description>Open the M1 mirror covers. The command is reported done right away, before the covers finish moving.</Description>
+        <Description>Open the M1 mirror covers. Note the command is reported done right away, before the covers finish moving.</Description>
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>ignored</EFDB_Name>
-            <Description/>
+	    <Description>True to open the 4 petals of the primary mirror cover</Description>
```

Comment:
Is this ignored or have to set it to `True`?

Reply:
According to the current agreement the description should be 

"Attribute required by the API, but is unused."

Reply:
Corrected to be consistent, except used "Attribute required by the SAL API, but is unused"